src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -154,26 +154,29 @@ const char* get_shader_override_type(const int shader_override_type)
       case 0:  return "no_override";
       case 1:  return "albedo";
       case 2:  return "ambient_occlusion";
-      case 3:  return "assembly_instances";
-      case 4:  return "barycentric";
-      case 5:  return "bitangent";
-      case 6:  return "coverage";
-      case 7:  return "depth";
-      case 8:  return "facing_ratio";
-      case 9:  return "geometric_normal";
-      case 10: return "materials";
-      case 11: return "object_instances";
-      case 12: return "original_shading_normal";
-      case 13: return "primitives";
-      case 14: return "ray_spread";
-      case 15: return "screen_space_wireframe";
-      case 16: return "shading_normal";
-      case 17: return "sides";
-      case 18: return "tangent";
-      case 19: return "uv";
-      case 20: return "world_space_position";
-      case 21: return "world_space_velocity";
-      case 22: return "world_space_wireframe";
+      case 3:  return "assemblies";
```

Comment:
We're breaking compatibility. I'm fine with that given that these are just diagnostic modes, but I just want to make sure we're aware of that :)

Reply:
Yep, I'm aware of this.